### DISCUSSION TITLE
Using local_path with WSL2 & Sail

### DISCUSSION THREAD

**Discussion by oliverhermanni** [spatie/laravel-ray#52] [Q&A]
Hi, 

I try to use laravel-ray with Sail and WSL2. So far it works fine. I also setup the config file and set the local_path of my project to `D:\Projects\Laravel\my_project`. Unfortunaly the "Goto Line" still is the  folder in Sails Docker container. 

Any help would be appreciated. :)

**Comment by hassanacto** [comment]
Yep, same here. We use Docker and I can't seem to get the navigation to IDE working. Set remote + local paths correctly.

**Reply by hassanacto** [reply]
Okay, so I've narrowed it down to the fact that it won't read the paths correctly from the env variables, but manually adding them in the ray.php configuration file seems to work for me.

**Reply by freekmurze** [reply]
@oliverhermanni Could you try the same?

**Reply by oliverhermanni** [reply]
@freekmurze The env()'s work for me. 

@hassanacto I've opened an issue with this problem and found a working solution, please check [this](https://github.com/spatie/laravel-ray/issues/61#issuecomment-758055499) comment, if you use WSL2 and PhpStorm.

**Reply by hassanacto** [reply]
@oliverhermanni ah okay, I'm on MacOS so I don't think that affects me. Hoping your issue is fixed soon!
